code/game/machinery/_machinery.dm

```diff
@@ -241,6 +241,8 @@ Class Procs:
 	else
 		if(interaction_flags_machine & INTERACT_MACHINE_REQUIRES_SILICON)
 			return FALSE
+		if(is_species(user, /datum/species/lizard/ashwalker))
```

Comment:
add an interaction_flags_machine flag, to allow ashwalkers so we can explicitly allow it if we need to, like with the silicons above?